regimes/mx/complements/fuel_complement.go

```diff
@@ -0,0 +1,58 @@
+// Package complements provides GOBL Complements for Mexican invoices
+package complements
+
+import (
+	"cloud.google.com/go/civil"
+	"github.com/invopop/gobl/bill"
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/num"
+	"github.com/invopop/gobl/org"
+	"github.com/invopop/gobl/tax"
+)
+
+// FuelComplement provides detailed information about fuel purchases made with
+// electronic wallets. In Mexico, e-wallet suppliers are required to report this
+// information as part of the invoices they issue to their customers.
+type FuelComplement struct {
+	// Customer's e-wallet account number
+	WalletAccountNumber string `json:"wallet-account-number" jsonschema:"title=Wallet Account Number"`
+	// List of fuel purchases made with the customer's e-wallets
+	Lines []*FuelLine `json:"lines" jsonschema:"title=Lines"`
+	// Summary of all the purchases totals, including taxes (calculated)
+	Totals bill.Totals `json:"totals" jsonschema:"title=Totals" jsonschema_extras:"calculated=true"`
```

Comment:
I'm reusing `bill.Totals` here but I'm not 100% sure about this as we only need the `Total` and `TotalWithTaxes` fields from that struct. It may be confusing for developers to have all the other fields available or not.

A penny for your thoughts!

Reply:
I'm inclined to agree... this strikes me as a new structure alongside the complements.

Reply:
I agree! Addressed in 4be0288